packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -1,45 +1,76 @@
 import { autorun } from 'mobx'
-import { addDisposer, getRoot, onAction, types } from 'mobx-state-tree'
+import { addDisposer, addMiddleware, getRoot, onAction, types } from 'mobx-state-tree'
+import { flatten } from 'lodash'
 
 import helpers from './feedback/helpers'
 import strategies from './feedback/strategies'
 
 const FeedbackStore = types
   .model('FeedbackStore', {
     isActive: types.optional(types.boolean, false),
-    rules: types.map(types.frozen({}))
+    rules: types.map(types.frozen({})),
+    showModal: types.optional(types.boolean, false),
+    allowSubjectAdvance: types.optional(types.boolean, false)
   })
+  .views(self => ({
+    get hideSubjectViewer () {
+      return flatten(Array.from(self.rules.values()))
+        .some(rule => rule.hideSubjectViewer)
```

Comment:
Why does hiding the subject viewer for one task hide it for the entire workflow? Or am I reading this line wrong?

Reply:
I think we intend to hide the SubjectViewer if just one rule has that attribute, overriding all other rules, or that's how we have [it in PFE now](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/features/feedback/classifier/helpers/open-feedback-modal.jsx#L36-L37). We'll want to make sure it's clear it'll override all other rules in the lab, if we keep it this way.